openfisca_us/parameters/benefit/CCDF/provider_type.yaml

```diff
@@ -0,0 +1,23 @@
+description: provider types
```

Comment:
we don't need parameters for this one, it'll just be an enum variable kind of like state_code

Reply:
I see. Thanks!